refresh/index.js

```diff
@@ -30,10 +31,75 @@ module.exports = generators.Base.extend({
     generators.Base.apply(this, arguments);
   },
 
+  constructor: function() {
+    generators.Base.apply(this,arguments);
+
+    // Allow the user to specify where the specification file is
+    this.option('specfile', {
+      desc: 'The location of the specification file.',
+      required: false,
+      hide: true,
+      type: String
+    })
+
+    this.option('spec', {
+      desc: 'The specification in a JSON format.',
+      required: false,
+      hide: true,
+      type: String
+    })
+  },
+
   initializing: {
-    ensureInProject: actions.ensureInProject,
```

Comment:
I think we should still check this **if** we are running the refresh generator directly (ie not composed with the app generator) and without a spec.

Note: if we make the app generator pass a spec instead of a config object, it will make this check simpler.